src/mscorlib/shared/System/IO/Path.Windows.cs

```diff
@@ -163,8 +163,8 @@ public static ReadOnlySpan<char> GetPathRoot(ReadOnlySpan<char> path)
         {
             ReadOnlySpan<char> result = new ReadOnlySpan<char>();
 
-            if (PathInternal.IsEffectivelyEmpty(path))
```

Comment:
You can now delete the IsEffectivelyEmpty override receiving a ReadOnlySpan that you added for Windows.